install_config/web_console_customization.adoc

```diff
@@ -68,6 +68,51 @@ link:https://github.com/openshift/origin-web-console/tree/master/extensions/exam
 Origin] repository on GitHub.
 ====
 
+[[setting-extension-properties]]
+=== Setting Extension Properties
+
+Extension scripts allow you to override the default behavior of the web console
+and customize it for your needs. For example, extension scripts can be used to
+add your own company's branding or to add company-specific capabilities. A
+common use case for this is rebranding or white-labelling for different
+environments. You can use the same extension code, but provide settings that
+change the web console.
```

Comment:
I feel like most of this paragraphe could get merged into the current Overview of this topic, since it's more generally about the what/why of using extensions overall. WDYT? And then this section could just start w/ the "If you have a specific extension" paragraph, since the section is mainly about setting those custom properties.